systemverilog-plugin/UhdmAst.cc

```diff
@@ -3824,6 +3824,12 @@ void UhdmAst::process_tf_call(AST::AstNodeType type)
             current_node->children.push_back(node);
         }
     });
+    // prefer full name of function
+    // this fixes case when we imports single function from package
```

Comment:
I think it should be `when we import a single function from a package`. Or I could suggest an impersonal alternative: `When a single function is imported from a package`

Reply:
Sure, changed.